1config-core/src/com/brunobonacci/oneconfig/util.clj

```diff
@@ -97,6 +97,15 @@
      (remove (where second :is? nil))
      (into {})))
 
+(defn required-names
+  "Returns required argument names separated by comma"
+  [name-map]
+  (->> name-map
+       (filter val)
+       (into {})
```

Comment:
there is no need to turn the sequence into a map just to extract the names
the sequence generated out of a map is a sequence of `[key value]` tuples
so to get the keys you can just use `first` example `(first [:env "dev"]) ;;=> :env`